app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,19 @@
+class ReviewsController < ApplicationController
+
+  before_action :require_user, only: [:create]
+
+  def create
+    @video = Video.find(params[:video_id])
+    @review = @video.reviews.new(params.require(:review).permit(:rating, :content).merge!(user: current_user))
+
+    if @review.save
+      flash[:notice] = "Review submitted"
```

Comment:
Be aware that the Bootstrap classes have changed to success and danger. The flashes partial handles this in this app, but if you are manually creating an alert you will need to use the new classes.